### PR TITLE
Add utilities for VM-Of-Today

### PR DESCRIPTION
**Pull request by lizzha**

### CODE REVIEW

File: Utilities/VmOfToday/Get-LatestLinuxNextVM.ps1 [hunk 1]
```diff
@@ -0,0 +1,232 @@
+﻿##############################################################################################
+# Get-LatestLinuxNextVM.ps1
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.SYNOPSIS
+	This script gets the latest linux-next VM, and add RG lock for the user
```
Comment (LiliDeng) [inline Utilities/VmOfToday/Get-LatestLinuxNextVM.ps1]: Seems this file name not reflect its real function, it adds locks for the latest linux-next VM with the user name

File: Utilities/VmOfToday/Get-LatestLinuxNextVM.ps1 [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+param
+(
+	[string] $AzureSecretsFile,
+	[string] $UserName,
+	[string] $DatabaseTableName = "VM_Of_Today",
+	[string] $LogFileName = "GetLatestLinuxNextVM.log"
+)
+
+Function Get-DatabaseRecord($secrets, $query) {
+	$resultTable = $null
```
Comment (LiliDeng) [inline Utilities/VmOfToday/Get-LatestLinuxNextVM.ps1]: This function defined many times, can we put all functions into one file, then use different functions to reach the goal, add/release lock for RG etc.

File: Utilities/VmOfToday/Release-LockedLinuxNextVM.ps1 [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+	Write-LogInfo "Removing the azure resource lock on resource group $($vmInfo.ResourceGroupName)"
+	Remove-AzureRmResourceLock -LockName $userName -ResourceGroupName $vmInfo.ResourceGroupName -Force
+
+	if ([string]::IsNullOrEmpty($vmInfo.UsedBy)) {
+		Write-LogWarn "UsedBy column is empty for resource group $($vmInfo.ResourceGroupName)"
+		return
+	}
+	$usedBy = $vmInfo.UsedBy.Replace($userName, "").Trim(",").Replace(",,", ",")
+	Write-LogInfo "Updating the record in database"
+	$SQLQuery = "update $DatabaseTableName set UsedBy='$usedBy' where ID =$($vmInfo.ID)"
```
Comment (LiliDeng) [inline Utilities/VmOfToday/Release-LockedLinuxNextVM.ps1]: Should here update UsedBy into empty?

File: Utilities/VmOfToday/Get-LatestLinuxNextVM.ps1 (hunk 2 above)
Comment (lizzha) [reply Utilities/VmOfToday/Get-LatestLinuxNextVM.ps1]: good point, will update

File: Utilities/VmOfToday/Release-LockedLinuxNextVM.ps1 (hunk 3 above)
Comment (lizzha) [reply Utilities/VmOfToday/Release-LockedLinuxNextVM.ps1]: each user should only release the lock added for him/her

**Comment by lizzha** [comment]
Close this one due to design change

**Pull request closed without merging**